packages/test-utils/src/wrapper.js

```diff
@@ -657,6 +657,17 @@ export default class Wrapper implements BaseWrapper {
           `is not defined on the component`
         )
       }
+      if (
+        typeof data[key] === 'object' &&
+        data[key] !== null &&
+        // $FlowIgnore : Problem with possibly null this.vm
+        data[key] === this.vm[key]
+      ) {
+        throwError(
+          `wrapper.setProps() called with ${key} property ` +
```

Comment:
Please reword:
`wrapper.setProps() called with the same object of the existing ${key} property. You must call `wrapper.setProps()` with a new object to trigger reactivity`